src/main/webapp/signUp.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Sign Up</title>
+    <link rel="stylesheet" href="style/style.css">
+</head>
+<body>
+    <div class="mainWrapper">
+        <div class="welcomeWrapper">
+            <h1>Sign Up</h1>
+        </div>
+        <div class="signWrapper">
+
+                    <form class="signInputWrapper" action="/signup" method="POST">
+                        <input type="text" id="username" name="username" placeholder="Username" required>
+                        <input type="email" id="email" name="email" placeholder="Email" required>
+                        <input type="password" id="password" name="password" placeholder="Password" required>
+                        <input value="Sign Up" type="submit" class="signButton" style="margin:0px;width:100%" onclick="goToMainPage()">
+                    </form>
```

Comment:
To make things easier, let's only offer sign-ups and logins through the google API. So this shouldn't be needed (we shouldn't need to store any passwords).